Add documentation for Suggested Actions on Non-Visual Elements

thsparks opened the pull request:
This adds a page for the missing documentation on non-visual elements. The link to these docs was added in a previous change, but the actual file was missing. Fixes #36.

mgoertz-msft reviewed (approved):
This looks good. However, how would the user know that this is only available for 16.9? Is this marked somewhere else together with the suggested actions API?

mgoertz-msft commented on documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility-nonvisualelements.md:
```diff
@@ -0,0 +1,23 @@
+    {
+        // Find the first visual parent of the non-visual view-item.
+        ModelItem parent = modelItem.Parent;
+        while(!parent.View.IsVisible)
```
Add a space to `while (...)`

thsparks commented:
> This looks good. However, how would the user know that this is only available for 16.9? Is this marked somewhere else together with the suggested actions API?

Good point. I've added a comment at the beginning of the file to indicate when we added support.
